xskillscore/tests/test_metric_results_accurate.py

```diff
@@ -74,12 +78,10 @@ def test_xs_same_as_skl_same_name(a_1d, b_1d, request):
     assert np.allclose(actual, expected)
 
 
-@pytest.fixture(params=["mean_absolute_percentage_error"])
-def test_xs_same_as_skl_with_zeros(a_1d_with_zeros, b_1d, request):
-    xs_metric, skl_metric = (
-        getattr(xs, request.param),
-        getattr(sklearn.metrics, request.param),
-    )
+@pytest.mark.parametrize("xs_skl_metrics", xs_skl_metrics_with_zeros)
+def test_xs_same_as_skl_with_zeros(a_1d_with_zeros, b_1d, xs_skl_metrics):
```

Comment:
nice, this should have failed for mape before

Reply:
yeah I think so